protocol/git/README.md

```diff
@@ -106,6 +106,7 @@ View a list of new commits. View changed files. Merge branch into master.
     git log origin/master..<branch-name>
     git diff --stat origin/master
     git checkout master
+    git pull
```

Comment:
The default `pull` action is to do a merge, whilst we use a rebase workflow.

Also, [above] we already prompt the user to fetch and update with `origin/master`. Which makes me wonder, is this step required?

[above]: https://github.com/thoughtbot/guides/pull/425/files#diff-1cc33e156f1a96c475b181ed56b14931R95